jest-common/src/main/java/io/searchbox/cluster/TasksInformation.java

```diff
@@ -0,0 +1,42 @@
+package io.searchbox.cluster;
+
+import io.searchbox.action.GenericResultAbstractAction;
+
+/**
+ * Currently only supports retrieving information for a particular task
+ */
+public class TasksInformation extends GenericResultAbstractAction {
+
+    protected String taskId;
+
+    protected TasksInformation(Builder builder) {
+        super(builder);
+        taskId = builder.taskId;
+        setURI(buildURI());
+    }
+
+    @Override
+    protected String buildURI() {
+        return super.buildURI() + "_tasks/" + taskId;
+    }
+
+    @Override
+    public String getRestMethodName() {
+        return "GET";
+    }
+
+    public static class Builder extends GenericResultAbstractAction.Builder<TasksInformation, Builder> {
+
+        protected String taskId;
+
+        public Builder(String taskId) {
```

Comment:
Getting this param as constructor argument will make it difficutl to add the other parameters to `_tasks` in the future. I would add using a method named `task(int taskId)`. So in future we can add have `tasks(int... tasksId)` Without specific taskId it will fetch all tasks.